solr/core/src/test/org/apache/solr/filestore/TestDistribFileStore.java

```diff
@@ -246,7 +246,7 @@ public static <T extends NavigableObject> T assertResponseValues(
   public static NavigableObject assertResponseValues(
       int repeats, SolrClient client, SolrRequest<?> req, Map<String, Object> vals)
       throws Exception {
-    Callable<NavigableObject> callable = () -> req.process(client);
+    Callable<NavigableObject> callable = () -> client.request(req);
```

Comment:
switched to request method, which returned NamedList which is a NaviagableObject.  `process` now has unbound type and thus could be anything.